feat: create template QA env deploiement (#17)
arkhn/.github: .github/ISSUE_TEMPLATE/QA_env_deployment.md

# Description 
<!-- Describe the particularity of this env if necessary -->

> # Test acceptance (Mandatory)

Reviewer: ```suggestion
# Test acceptance
<!-- Mandatory -->
```

Infra team:

> - [] How to connect is create

Reviewer: ```suggestion
- [] The "How to connect" page is created
```

> - [] Presence of necessary passwords in bitwarden

Reviewer: I think it would be usefull to list the passwords to set in Bitwarden.

- [] Check if at least 2 people can work at the same time

> 

Reviewer: - verify we can connect to the main web components from the Bastion or the dev workstation:
  - admin
  - dagit
  - cloudbeaver
  - keycloak (admin)
- verify the metrics are pushed in AZmonitoring
- verify the TLS is configured (no auto-signed certificate)

Data team:

> - [] Connexion to the plateform has been tested

Reviewer: ```suggestion
- [] Connection to the platform has been tested
```

# Description 
<!-- Describe the particularity of this env if necessary -->

> # Test acceptance

Reviewer 2: ```suggestion
# Acceptance tests
```

<!-- Mandatory -->

Infra team:
- [] The "How to connect" page is created
- [] Presence of necessary passwords in bitwarden:
    - argocd
    - keycloak
    - code-server
    - clickhouse datalake
    - cloudbeaver
    - vault token
    - minio

> - [] Check if at least 2 people can work at the same time

Reviewer 2: ça veut dire quoi précisément ? Genre à Besançon ce serait pas bon du coup ?

Reviewer: Maintenant, ça l'est. Mais oui, c'est l'exemple. Au début on ne pouvait pas se connecter à 2.

- [] Verify we can connect to the main web components from the Bastion or the dev workstation:
    - admin
    - dagit
    - cloudbeaver
    - keycloak (admin)
- [] Verify the metrics are pushed in AZmonitoring
- [] Verify the TLS is configured (no auto-signed certificate)

> - [] Create the repository dagster-code-besancon on github

Reviewer 2: ```suggestion
- [] Create the corresponding repository dagster-code- on dockerhub
```

> - [] Create arkhn-platform-dagster-user-deployments-k8s-dagster-code on argocd

Reviewer 2: Aucune idée de ce que c'est censé être le `arkhn-platform-dagster-user-deployments-k8s-dagster-code`

Reviewer: C'est deployment de dagster code. En théorie, il suffit de faire un sync sur ArgoCD. Je ne l'avais pas fait à Besançon car je savais que l'image n'existait pas. Mais fait partie du process normal, je ne sais pas s'il faut l'ajouter à la liste.

Author: Ok je supprime
